docsource/modules140-150.rst

```diff
@@ -670,6 +714,82 @@ Module coverage 14.0 -> 15.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | survey                                          |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
+| test_access_rights                              |                      |No DB layout changes.                            |
```

Comment:
Test modules shouldn't appear here

Reply:
why?

Reply:
Because they are noise and they are not installed in real databases. The script that populates this table should filter out them.